pkg/ccl/changefeedccl/encoder_test.go

```diff
@@ -1051,3 +1051,31 @@ func makeTestTypes(rng *rand.Rand, numKeyTypes, numColTypes int) (keyTypes, valT
 
 	return randTypes(numKeyTypes, true), randTypes(numColTypes, false)
 }
+
+func TestParquetEncoder(t *testing.T) {
```

Comment:
I would love to see at least some of the tests in cdc_test (Basic test for example), integrated with parquet format.
I really don't see any reason why it shouldn't work;
I understand that perhaps handling of null might break test expectations -- but that can be easily fixed by test feed -- we have the metadata, so, you can set any missing columns to null...
?